flutter/lib/src/sentry_flutter.dart

```diff
@@ -60,8 +60,6 @@ mixin SentryFlutter {
     SentryFlutterOptions options,
     MethodChannel channel,
   ) async {
-    options.debug = kDebugMode;
```

Comment:
I believe removing this would introduce a bug now, after deep thinking, we should do this in both packages first thing.

otherwise all the logging being done before this flag is set in the sentry package will be still `false` when it should be enabled.
actually, this bug even already exists I guess, here we should also do this:

https://github.com/getsentry/sentry-dart/blob/main/dart/lib/src/sentry.dart#L199-L202

otherwise even if `debug=true`, there's no logging at all (its a hunch)

Reply:
In Sentry it is already the first thing we do. But yeah in sentry_flutter it makes sense to do as first thing.
I'm not sure though how I would go about testing this.

Reply:
I created an issue for this here: https://github.com/getsentry/sentry-dart/issues/384